packages/vscode-extension/WHATISNEW.md

```diff
@@ -10,6 +11,20 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+# Changelog
+
+## 3.6.0 - Mar 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- User can use Teams Toolkit to create CI/CD pipelines for GitHub, Azure DevOps and Jenkins.
```

Comment:
Align with the following tooltip would be better.
`Create workflow automation templates for Github, Azure DevOps and Jenkins`
We're not going to help users create pipeline, just workflow/pipeline templates in yml/jenkins files.